index.src.html

```diff
@@ -830,18 +870,28 @@ <h4 id="algorithm-create" algorithm>Create a `Credential`</h4>
             types in order to support a "sign-up" use case. For the moment, though, we're punting
             on that by restricting the dictionary to a single entry.
 
-    5.  Let |p| be [=a new promise=].
+    6.  Let |p| be [=a new promise=].
 
-    6.  Run the following steps [=in parallel=]:
+    7.  Let |origin| be |settings|'s [=environment settings object/origin=].
 
-        1.  Let |r| be the result of executing |interfaces|[0] {{Credential/[[Create]](options)}}
-            internal method on |options|.
+    8.  Run the following substeps [=in parallel=]:
 
-        2.  If |r| is an [=exception=], [=reject=] |p| with |r|.
+        1.  Let |r| be the result of executing |interfaces|[0]'s {{Credential/[[Create]](origin, options)}}
+            internal method given |origin| and |options|.
 
-            Otherwise, [=resolve=] |p| with |r|.
+        2.  If |r| is an [=exception=] or `null`, [=reject=] |p| with |r|, and terminate these substeps.
```

Comment:
The current behavior is to [=resolve=] with null instead of rejecting with it. I might argue that's the wrong default, but let's not change it in this PR.

Reply:
oh, ok.